test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -237,17 +218,18 @@ func (na nodeAffinity) mutatePodTemplate(pod *v1.Pod) {
 // generateNodes generates nodes to be used for scheduling.
 func (inputConfig *schedulerPerfConfig) generateNodes(config *testConfig) {
 	for i := 0; i < inputConfig.NodeCount; i++ {
-		config.schedulerSupport.Client.CoreV1().Nodes().Create(config.mutatedNodeTemplate)
+		config.clientset.CoreV1().Nodes().Create(config.mutatedNodeTemplate)
```

Comment:
It seems we rely on `testConfig.clientset`, so we can:

- put `factory.Config` in `testConfig`
- Or, put both `podInformer` and `clientset` in `testConfig` if we plan to eliminate `factory.Config` in the future

Reply:
done